apps/sms/js/information.js

```diff
@@ -365,7 +378,12 @@ var Information = function(type) {
 
   var prefix = 'information-' + this.name;
   this.container = document.getElementById(prefix);
-  this.parent = document.getElementById('thread-messages');
+  if(this.name == 'participants'){
+    this.parent = document.getElementById('information-group-view');
```

Comment:
The "parent" here didn't have much meaning here once you split into 2 different panels instead of thread container. I think the reasonable adjustment here is fetch the `section` element as this.panel first, and set all the elements by this.panel.querySelector (and put container in elements). In show/hide we only need to change the panel's classlist